### ISSUE TITLE
eCheck-in: Add Link to Access the Upcoming Appointments Page

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#89066]
## Wireframes
[Wireframes](https://www.figma.com/design/7Ib7RxiIC4QB53FDBO2a8c/Unified-check-in-%7C-PCI?node-id=1887-14128&t=VfR5cZYPsv2WOg9p-0)

## Tasks
- Add a link on the eCheck-in Appointments What to Do Next page (see wireframes)
- Add a link on the eCheck-in You're Checked In page (see wireframes)

**Comment by loripusey** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/89066?planning-poker) with Zenhub @brianseek @wullaski

**Comment by brianseek** [comment]
I don't think we can point until we know more about which things we can support. We have a spike in this sprint for that part.

**Comment by loripusey** [comment]
@brianseek @wullaski can we point this one yet?

**Comment by brianseek** [comment]
@loripusey not yet
